pkg/defaults/init.go

```diff
@@ -0,0 +1,49 @@
+package defaults
+
+import (
+	"github.com/port-labs/port-k8s-exporter/pkg/port"
+	"github.com/port-labs/port-k8s-exporter/pkg/port/cli"
+	"github.com/port-labs/port-k8s-exporter/pkg/port/integration"
+	"k8s.io/klog/v2"
+)
+
+func getEventListenerConfig(eventListenerType string) *port.EventListenerSettings {
+	if eventListenerType == "KAFKA" {
+		return &port.EventListenerSettings{
+			Type: eventListenerType,
+		}
+	}
+	return nil
+}
+
+func InitIntegration(portClient *cli.PortClient, applicationConfig *port.Config) error {
+	existingIntegration, err := integration.GetIntegration(portClient, applicationConfig.StateKey)
+	defaultIntegrationConfig := &port.IntegrationAppConfig{
+		Resources: applicationConfig.Resources,
+	}
+
+	if err != nil {
+		if defaultIntegrationConfig.Resources == nil && applicationConfig.CreateDefaultResources {
+			if err := initializeDefaults(portClient, applicationConfig); err != nil {
+				klog.Warningf("Error initializing defaults: %s", err.Error())
```

Comment:
not really because there might be a scenerio that you want a second k8s exporter, there are already default blueprints, and there is no integration

in this scenario i want the integration to just create an integration with empty config

that's what happening in ocean so far

we also talked about this yesterday and agree to have this logic right now and maybe change it if we want

i can explain more in private if youd like me to